### ISSUE TITLE
关于方向分类的预测模型和推理模型

### ISSUE THREAD

**Issue by strWuYaoJu** [PaddlePaddle/PaddleOCR#1167]
使用训练得到的预训练模型进行预测，结果都正确，但是经过export_model.py导出得到的推理模型，却很多都不正确，差距很大。这里train.py和export_model.py使用的都是一样的configs 。

**Comment by LDOUBLEV** [comment]
export_model导出模型的时候，模型有没有加载上？

**Comment by strWuYaoJu** [comment]
@LDOUBLEV 执行  python3 tools/export_model.py -c configs/cls/cls_mv3.yml -o Global.checkpoints=./output/cls_mv3/best_accuracy Global.save_inference_dir=./inference/cls_inference，显示如下：

2020-11-13 10:57:32,746-INFO: {'Global': {'debug': False, 'algorithm': 'CLS', 'use_gpu': True, 'epoch_num': 200, 'log_smooth_window': 20, 'print_batch_step': 100, 'save_model_dir': 'output/cls_mv3', 'save_epoch_step': 50, 'eval_batch_step': 500, 'train_batch_size_per_card': 8, 'test_batch_size_per_card': 1, 'image_shape': [3, 200, 50], 'label_list': ['0', '180'], 'distort': True, 'reader_yml': './configs/cls/cls_reader.yml', 'pretrain_weights': None, 'checkpoints': './output/cls_mv3/best_accuracy', 'save_inference_dir': './inference/cls_inference', 'infer_img': None}, 'Architecture': {'function': 'ppocr.modeling.architectures.cls_model,ClsModel'}, 'Backbone': {'function': 'ppocr.modeling.backbones.rec_mobilenet_v3,MobileNetV3', 'scale': 0.35, 'model_name': 'small'}, 'Head': {'function': 'ppocr.modeling.heads.cls_head,ClsHead', 'class_dim': 2}, 'Loss': {'function': 'ppocr.modeling.losses.cls_loss,ClsLoss'}, 'Optimizer': {'function': 'ppocr.optimizer,AdamDecay', 'base_lr': 0.001, 'beta1': 0.9, 'beta2': 0.999, 'decay': {'function': 'cosine_decay', 'step_each_epoch': 1169, 'total_epoch': 200}}, 'TrainReader': {'reader_function': 'ppocr.data.cls.dataset_traversal,SimpleReader', 'num_workers': 8, 'img_set_dir': './train_data/cls/train', 'label_file_path': './train_data/cls/cls_gt_train.txt'}, 'EvalReader': {'reader_function': 'ppocr.data.cls.dataset_traversal,SimpleReader', 'img_set_dir': './train_data/cls/test', 'label_file_path': './train_data/cls/cls_gt_test.txt'}, 'TestReader': {'reader_function': 'ppocr.data.cls.dataset_traversal,SimpleReader'}}
W1113 10:57:33.814373 29242 device_context.cc:237] Please NOTE: device: 0, CUDA Capability: 70, Driver API Version: 10.1, Runtime API Version: 9.0
W1113 10:57:33.818773 29242 device_context.cc:245] device: 0, cuDNN Version: 7.3.
2020-11-13 10:57:35,489-INFO: Finish initing model from ./output/cls_mv3/best_accuracy
inference model saved in ./inference/cls_inference/model and ./inference/cls_inference/params
save success, output_name_list: ['decoded_out', 'predict']

感觉没啥问题呀

**Comment by WenmuZhou** [comment]
export之后，用predict_cls试一下

**Comment by strWuYaoJu** [comment]
@WenmuZhou 使用predict_cl执行得到的结果还是不对，跟infer_cls预测的结果差距很大

**Comment by WenmuZhou** [comment]
看看predict_cls的时候图片的shape
和infer下时候是否一致

**Comment by strWuYaoJu** [comment]
@WenmuZhou shape是一样的

**Comment by WenmuZhou** [comment]
你的图片是什么样的，我看你的shape是[3, 200, 50],感觉图片是竖直的

**Comment by strWuYaoJu** [comment]
@WenmuZhou 是我的图像长、宽写反了，修改后重新训练，可以了，谢谢！
